src/main/java/org/jabref/gui/edit/automaticfiededitor/copyormovecontent/CopyOrMoveFieldContentTabView.java

```diff
@@ -89,30 +90,34 @@ public Field fromString(String name) {
             }
         });
 
-        fromFieldComboBox.getSelectionModel().selectFirst();
-        toFieldComboBox.getSelectionModel().selectLast();
-
         viewModel.fromFieldProperty().bindBidirectional(fromFieldComboBox.valueProperty());
         viewModel.toFieldProperty().bindBidirectional(toFieldComboBox.valueProperty());
     }
 
+    private void populateFieldsComboBox() {
+        fromFieldComboBox.getItems().setAll(viewModel.getAllFields().stream().sorted(Comparator.comparing(Field::getDisplayName)).toList());
```

Comment:
Do that sorting already in the viewModel, so just something like setAll(viewModel.getallFieldSorted())